Voycar.Api.Web/Generic/Endpoint/Post/Single.cs

```diff
@@ -18,7 +18,16 @@ protected Single(IRepository<TEntity> repository, string[] roles)
     public override void Configure()
     {
         this.Post(typeof(TEntity).Name.ToLowerInvariant());
-        this.Roles(this.Roles);
+
```

Comment:
Warum genau wollen wir das so machen?

Reply:
Die Endpunkte `Plan/Get/all` und `Plan/Get/{id}` müssen auch für nicht eingloggte user erreichbar sein. Sonst kann man seinen Plan z.B. während der Registrierung nicht auswählen 😅 
Da diese Endpunkte ja einfach generic Endpunkte sind, brauchen wir in den generics Logik, um `AllowAnonymous` zu machen. 

Man könnte jetzt argumentieren, dass das ein Sicherheitsrisiko ist, als default (wenn man keine Rollen angibt) AllowAnonymous zu machen. Ansonsten sieht mir das aber sinnvoll aus.